### ISSUE TITLE
dotnet-install.sh is actually a powershell script?

### ISSUE THREAD

**Issue by zeroxia** [dotnet/vscode-dotnet-runtime#583]
I use VS Code installation on Windows, and do ssh-remote to a Linux container,
upon launch of VS Code, there is an error notification:

```
An error occurred while installing .NET (6.0): .NET Acquisition Failed: Installation failed: Error: Command failed: "/home/<username>/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh" -InstallDir "/home/<username>/.vscode-server/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/6.0.8" -Version 6.0.8 -Runtime dotnet
/home/<username>/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh: line 6: syntax error near unexpected token `newline'
/home/<username>/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh: line 6: `<#'

```

And as I check the file content:

```
$ head -n 50 '.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh'
#
# Copyright (c) .NET Foundation and contributors. All rights reserved.
# Licensed under the MIT license. See LICENSE file in the project root for full license information.
#

<#
.SYNOPSIS
    Installs dotnet cli
.DESCRIPTION
    Installs dotnet cli. If dotnet installation already exists in the given directory
    it will update it only if the requested version differs from the one already installed.
.PARAMETER Channel
    Default: LTS
    Download from the Channel specified. Possible values:
    - Current - most current release
    - LTS - most current supported release
    - 2-part version in a format A.B - represents a specific release
          examples: 2.0, 1.0
    - 3-part version in a format A.B.Cxx - represents a specific SDK release
          examples: 5.0.1xx, 5.0.2xx
          Supported since 5.0 release
    Note: The version parameter overrides the channel parameter when any version other than 'latest' is used.
.PARAMETER Quality
    Download the latest build of specified quality in the channel. The possible values are: daily, signed, validated, preview, GA.
    Works only in combination with channel. Not applicable for current and LTS channels and will be ignored if those channels are used.
    For SDK use channel in A.B.Cxx format: using quality together with channel in A.B format is not supported.
    Supported since 5.0 release.
    Note: The version parameter overrides the channel parameter when any version other than 'latest' is used, and therefore overrides the quality.
.PARAMETER Version
    Default: latest
    Represents a build version on specific channel. Possible values:
    - latest - the latest build on specific channel
    - 3-part version in a format A.B.C - represents specific version of build
          examples: 2.0.0-preview2-006120, 1.1.0
.PARAMETER Internal
    Download internal builds. Requires providing credentials via -FeedCredential parameter.
.PARAMETER FeedCredential
    Token to access Azure feed. Used as a query string to append to the Azure feed.
    This parameter typically is not specified.
.PARAMETER InstallDir
    Default: %LocalAppData%\Microsoft\dotnet
    Path to where to install dotnet. Note that binaries will be placed directly in a given directory.
.PARAMETER Architecture
    Default: <auto> - this value represents currently running OS architecture
    Architecture of dotnet binaries to be installed.
    Possible values are: <auto>, amd64, x64, x86, arm64, arm
.PARAMETER SharedRuntime
    This parameter is obsolete and may be removed in a future version of this script.
    The recommended alternative is '-Runtime dotnet'.
    Installs just the shared runtime bits, not the entire SDK.
```

**Comment by zeroxia** [comment]
This is interesting, after I reload the ssh-remote window, the script become valid bash script on remote Linux system.

Then I quit all VS Code windows, and reopen VScode with a windows local workspace folder, there is a similar error notification, showing that the `dotnet-install.ps1` is actually a bash script.

![capture error notification](http://tva1.sinaimg.cn/large/61260048gy1h5ws8c90vdj215j0gwwuq.jpg)

I can now reload this VS Code window and the script is normal, as I can see it is downloading the runtime in the output window:

> Downloading the .NET Runtime.
> Downloading .NET version(s) 6.0.8 .........

* .NET Install Tool for Extension Authors: v1.5.0
* vs code version:
```
Version: 1.71.0 (user setup)
Commit: 784b0177c56c607789f9638da7b6bf3230d47a8c
Date: 2022-09-01T07:36:10.600Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No
```

Error message:

```
An error occurred while installing .NET (6.0): .NET Acquisition Failed: Installation failed: At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:25 char:3
+ if [ -t 1 ] && command -v tput > /dev/null; then
+   ~
Missing '(' after 'if' in if statement.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:25 char:5
+ if [ -t 1 ] && command -v tput > /dev/null; then
+     ~
Missing type name after '['.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:25 char:13
+ if [ -t 1 ] && command -v tput > /dev/null; then
+             ~~
The token '&&' is not a valid statement separator in this version.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:28 char:7
+     if [ -n "$ncolors" ] && [ $ncolors -ge 8 ]; then
+       ~
Missing '(' after 'if' in if statement.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:28 char:9
+     if [ -n "$ncolors" ] && [ $ncolors -ge 8 ]; then
+         ~
Missing type name after '['.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:28 char:26
+     if [ -n "$ncolors" ] && [ $ncolors -ge 8 ]; then
+                          ~~
The token '&&' is not a valid statement separator in this version.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:28 char:30
+     if [ -n "$ncolors" ] && [ $ncolors -ge 8 ]; then
+                              ~
Missing type name after '['.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:42 char:13
+ say_warning() {
+             ~
An expression was expected after '('.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:43 char:72
+ ... printf "%b\n" "${yellow:-}dotnet_install: Warning: $1${normal:-}" >&3
+                                                                        ~
Missing file specification after redirection operator.
At C:\Users\<user.name>\.vscode\extensions\ms-dotnettools.vscode-dotnet-runtime-1.5.0\dist\install 
scripts\dotnet-install.ps1:43 char:72
+ ... printf "%b\n" "${yellow:-}dotnet_install: Warning: $1${normal:-}" >&3
+                                                                        ~
The ampersand (&) character is not allowed. The & operator is reserved for future use; wrap an ampersand in double 
quotation marks ("&") to pass it as part of a string.
Not all parse errors were reported.  Correct the reported errors and try again.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : MissingOpenParenthesisInIfStatement
  ```

**Comment by nagilson** [comment]
Heya, thanks for the detailed issue and image. This is a dupe of https://github.com/dotnet/vscode-dotnet-runtime/issues/159 and will be fixed in the 1.6.1 release. Closing, but feel free to ask any concerns over there.
